test/e2e/policies.test-suite/balloons/n4c32/test01-dynamic-baloons/code.var.sh

```diff
@@ -0,0 +1,83 @@
+terminate cri-resmgr
+cri_resmgr_cfg=${TEST_DIR}/balloons-dynamic.cfg cri_resmgr_extra_args="-metrics-interval 4s" launch cri-resmgr
+
+# pod0-pod7: create 8 balloons, where each lands on a different NUMA node.
+# Each balloon (except one that lands on the NUMA node with reserved CPUs)
+# has 1 shared CPU at the most since a NUMA node has 4 CPUs and a pod is
+# requesting 1 CPU. Only one of the balloon that using NUMA node with
+#reserved CPU has 0 shared CPUs.
+CPUREQLIM="3"
+INITCPUREQLIM="100m-100m 100m-100m 100m-100m"
+POD_ANNOTATION="balloon.balloons.cri-resource-manager.intel.com: dynamic"
+n=8 create multicontainerpod
+sleep 5
```

Comment:
I have reservation about the forced sleeps, they just delay the progress of the test without much benefit. I think `verify-metrics-has-line` will try the fetch the stuff multiple times anyway. If not possible to remove the sleep, it should be made smaller.

Reply:
During local runs of these tests, I noticed that some metrics (like deletion of a pod and CPUs of it is becoming free) didn't get updated fast enough (default -metrics-interval 4 seconds) that caused some verification to fail. And similar issue was discovered by @askervin during his trials. And I just realized after your comment, that we don't need sleep for those checks that doesn't use verify-metrics-has-no-line and only verify-metrics-has-line. But for checks with verify-metrics-has-no-line we can either sleep here as it is now (should be more than default interval 4 secs) or I can add a timeout to verify-metrics-has-no-line as we have for [verify-metrics-has-no-line](https://github.com/intel/cri-resource-manager/blob/8a8ebb63b3e199ded81bd0e355b0d2ac56036d86/test/e2e/policies.test-suite/balloons/verify.source.sh#L7). WDYT?

Reply:
@fmuyassarov , +1 from me to adding the timeout to verify-metrics-has-no-line, and getting rid of sleeps.

Reply:
Adding timeout to verify-metrics func sounds good, +1 to that.

Reply:
done